src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor().name();
+        int number = new NumberSupplier().getRandomNumber();
```

Comment:
lets make ColorSupplier and NumberSupplier global, because you are creating a new instance of those classes each time this method is invoked